.gitignore

```diff
@@ -0,0 +1,16 @@
+﻿################################################################################
+# This .gitignore file was automatically created by Microsoft(R) Visual Studio.
+################################################################################
+
+/.vs/LoginDemo
+/.vs
+/LoginDemo/.vs
+/LoginDemo/LearningSelenium/bin/Debug/net7.0
+/LoginDemo/LearningSelenium/obj
+/LoginDemo/LearningSelenium/GlobalUsings.cs
+/LoginDemo/LearningSelenium/LearningSelenium.csproj
```

Comment:
We usually push such files as .csproj and .sln(in case if reviewers will need to run it)

Reply:
.csproj file pushed successfully